### ISSUE TITLE
Adding transgender icon symbol

### ISSUE THREAD

**Issue by ciacicode** [phosphor-icons/homepage#452]
It would be good to include a transgender symbol icon for inclusion,  right now there are only the binary gender symbols.

![Gender_sign_(bold)](https://github.com/phosphor-icons/homepage/assets/6615030/dea8f7a5-1415-43cb-947a-8b8bb8e6bdd2)

**Comment by rektdeckard** [comment]
You are mistaken. We have icons for `gender-transgender`, `gender-neuter`, `gender-intersex`, and `gender-nonbinary`.
